include/z64.h

```diff
@@ -1103,7 +1103,7 @@ typedef struct {
     /* 0x14 */ u32 put;
     /* 0x18 */ u8 unk_18[0x20-0x18];
     /* 0x20 */ u8 data[0x10000-0x20];
-} ISVDbg;
+} ISVDbg; // size = 0x10000
```

Comment:
Probably doesn't matter but this whole struct will most likely be removed in #1468.